### DISCUSSION TITLE
I try this possibility nothie worked. still, this error is not gone

### DISCUSSION THREAD

**Discussion by umer310** [gulpjs/gulp#2639] [Help]
`npm install --global gulp-cli`
`npm init`
`npm install --save-dev gulp`
`gulp --version`
 After running all these commend on cmd then this error. Try many other simulation like update node path reinstalled node update node and npm nothing is worked,
`gulp : The term 'gulp' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling 
of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ gulp -v
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (gulp:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException`

Before you open this issue, please complete the following tasks:

* [ ] use the search bar at the top of the page to search this repository for similar issues or discussions that have already been opened.
* [ ] if you are looking for help from the gulp team or community, open a [discussion](https://github.com/gulpjs/gulp/discussions).
* [ ] if you think there is a problem with the plugin you're using, open a [discussion](https://github.com/gulpjs/gulp/discussions).
* [ ] if you think there is a bug in our code, open this issue.

### Please give us a sample of your gulpfile

```js
var gulp = require(‘gulp');

gulp.task(function () {

});
```

### Terminal output / screenshots

```sh
$ gulp
```

### Please provide the following information:
* OS & version [e.g. MacOS Catalina 10.15.4]:
* node version (run `node -v`):
* npm version (run `npm -v`):
* gulp version (run `gulp -v`):

**Comment by phated** [comment]
Our issue tracker is not a support forum. I will transfer this to a Discussion, which is the place you are supposed to ask support questions.
